cpp/Platform.Random.Tests/RandomExtensionsTests.cpp

```diff
@@ -1,47 +1,45 @@
-﻿#include <Platform.Random.h>
-#include <gtest/gtest.h>
-
-namespace Platform::Random::Tests
+﻿namespace Platform::Random::Tests
 {
     TEST(RandomExtensionsTests, NextUInt64Test)
     {
         auto lastValue = 0UL;
         auto theSameCount = 0;
-        for (auto i = 0; i < 10; i++)
+        for (auto i = 0; i < 10; ++i)
```

Comment:
Are you a believer?

Reply:
With today compilers there two ways are exactly the same. Compilers are not insane to make one of them faster. Am I wrong?

Reply:
They is the [same](https://godbolt.org/z/xKcn78cb9)